src/gevent/monkey.py

```diff
@@ -357,11 +367,13 @@ def _patch_module(name,
                   _warnings=None,
                   _notify_will_subscribers=True,
                   _notify_did_subscribers=True,
-                  _call_hooks=True):
+                  _call_hooks=True,
+                  _patch_module=False,
+                  _package_prefix='gevent.'):
```

Comment:
In the tradition of 'explicit is better than implicit', it would probably be better to make this a required argument instead of defaulting.

Instead of an extra argument, what if `name` was required to be the fully-qualified name instead of the simple module name?  That would allow patches that consist only of a module, not a package. (With a small amount of work, that could generalize to allowing arbitrary objects to be the patch source.)

Reply:
I have no problem with any aesthetic API changes (argument requirement, order, naming, refactoring).